src/components/jurisdiction/Jurisdiction.stories.ts

```diff
@@ -16,3 +16,14 @@ export const Default = Template.bind({})
 Default['args'] = {
   showUsaJurisdictions: false
 }
+
+export const JurisdictionInitialValue = Template.bind({})
+JurisdictionInitialValue['args'] = {
+  showUsaJurisdictions: false,
+  initialValue: {
+    group: 0,
+    text: 'Alberta',
+    value: 'AB',
+    separator: false
+  }
```

Comment:
There's information in here that you won't be saving to or restoring from a draft.

I think you need to convert the initial value from `{ country, region }` (which is what the parent component provides) to { group, value }` (which is what this component understands).

Reply:
Yes. You are right. Looking into it.